Objects/Objects/GIS/VectorLayer.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using Speckle.Core.Models;
+
+namespace Objects.GIS;
+
+public class VectorLayer : Collection
+{
+  public CRS? crs { get; set; }
+  public string? units { get; set; }
+  public Base? attributes { get; set; }
+  public string? geomType { get; set; }
+  public Dictionary<string, object>? renderer { get; set; }
```

Comment:
Same nullability question, can a vector layer have everything be null?

Reply:
regarding all nullability comments: these are the copies from specklepy, not sure we should specify them differently in core